Brightness Sync/AppDelegate.swift

```diff
@@ -4,79 +4,98 @@ import os
 class AppDelegate: NSObject, NSApplicationDelegate {
     let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.squareLength)
     let statusIndicator = NSMenuItem(title: "Starting", action: nil, keyEquivalent: "")
-    
+
+    let updateInterval = 0.1
     var syncTimer: Timer?
-    
+
     var lastBrightness: Double?
-    
+
+    let brightnessOffsetKey = "BSBrightnessOffset"
+    var brightnessOffset: Double {
```

Comment:
Perhaps having a different default value than 0 would be more appropriate. I'm using about -20% / - 25% to match up my MBP and the LG. On the other hand, this means the display is automatically deprived of it's max brightness, which somebody might not like. So 0 might be the safer option.

Reply:
I'll go with 0. This should also be the correct setting for users still on Mojave.